styles.css

```diff
@@ -0,0 +1,316 @@
+@media (min-width: 375px) and (max-width: 768px) {
```

Comment:
- [ ] Please I kindly suggest sticking to [FIGMA DESIGN TEMPLATE](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=67-1095&t=L3FgpgXxWWpBKXxA-0). 😃 

_**REQUIRED FIGMA DESIGN**_


![Screenshot (157)](https://user-images.githubusercontent.com/78104544/229080229-5df508e6-b100-43eb-abf7-7158b2e1f6c5.png)

![Screenshot (153)](https://user-images.githubusercontent.com/78104544/229078509-e51aa9e3-1120-4674-82b6-b27e30b68623.png)

_**YOUR DESIGN**_

- [ ] Please ensure that the content fits the whole `width` of the screen. 

![Screenshot (156)](https://user-images.githubusercontent.com/78104544/229082024-a203157c-5d01-4de8-8d8d-9c791b7ff62d.jpg)

![Screenshot (161)](https://user-images.githubusercontent.com/78104544/229082601-086febd2-60a6-428f-8833-b25066cf4963.jpg)


![Screenshot (160)](https://user-images.githubusercontent.com/78104544/229081483-74db43f1-1807-430b-b084-f8687cbeba1d.jpg)